OrbitGl/CaptureWindow.cpp

```diff
@@ -832,6 +839,11 @@ void CaptureWindow::RenderUI() {
     m_StatsWindow.AddLine(VAR_TO_STR(m_WorldMinWidth));
     m_StatsWindow.AddLine(VAR_TO_STR(m_MouseX));
     m_StatsWindow.AddLine(VAR_TO_STR(m_MouseY));
+    m_StatsWindow.AddLine(VAR_TO_STR(total_draw_count_));
+    if (total_draw_count_ != 0) {
+      double avg_draw_time = total_draw_time_secs_ / static_cast<double>(total_draw_count_);
```

Comment:
I think a moving average would be better here, we'd be able to visualize spikes a bit better.  It's not essential, but maybe we should at least reset total_draw_time_secs_ on the start of a capture?  Also, I was thinking about introspection for the UI side, I don't think it's a good idea in the end since we won't have the scheduling info.  I think the best approach would be to make it really easy to spawn a new instance of Orbit dedicated to profiling Orbit.

Reply:
Removed instrumentation code.